diagnostics/networking_diagnostics.bat

```diff
@@ -0,0 +1,37 @@
+set networking_folder=".\networking_logs"
+set neworking_logs_zip=".\WslNetworkingLogs.zip"
+
+mkdir %networking_folder%
+
+wsl.exe tr -d "\r" ^| bash < ./networking.sh > %networking_folder%\wsl_network_configuration_before.log
+powershell Get-NetRoute > %networking_folder%\get_netroute.log
+
+powershell invoke-expression 'cmd /c start powershell -Command { .\collect-wsl-logs.ps1 }'
+powershell invoke-expression 'cmd /c start powershell -Command { "wsl.exe sudo tcpdump -n -i any > tcpdump.log" }'

```

Comment:
Let's not call sudo and use wsl -u root instead

Reply:
fixed in iteration 3